### ISSUE TITLE
Support for multi-module projects

### ISSUE THREAD

**Issue by ceedubs** [ceedubs/sbt-ctags#4]
Multi-module projects seem like they shouldn't be terribly hard to support, but currently the plugin doesn't handle them.

**Comment by adelbertc** [comment]
@quintona Any updates on this? :-) Was about to look into adding support myself, but looks like you've beat me to it

**Comment by adelbertc** [comment]
How does this look? https://github.com/adelbertc/sbt-ctags/commit/b2d1694e1f1a087c5fe1cc3f5e668dbdd595aac8

Tried it on some small repos locally, seems to work. Issue now is if sub-project A depends on B, the generated tags for A doesn't include B so jump-to-definition doesn't like it, though that was discussed in the [original PR](https://github.com/ceedubs/sbt-ctags/pull/7).

> in the worst-case scenario we can throw together a shell script to do aggregation if needed.

**Comment by ceedubs** [comment]
@adelbertc that looks promising. Thanks! It will probably be a little while before I get around to giving it a try. Hopefully I can make the time this weekend.

Yeah, having A not pick up B's tags is a bummer. It sounds like your commit could be a step in the right direction, though. And maybe with time we can get that bit sorted out.

**Comment by adelbertc** [comment]
If we dumped all the tag files in the root project directory w/ different names, it's easier to collect them all using something like [this](http://stackoverflow.com/questions/12916743/include-ctags-files-recursively-from-a-directory), as you mentioned in the PR. I experimented with this a little bit by simply prepending each project's `tag` file with the `id`.

Not sure how good/bad that approach is.

**Comment by aloiscochard** [comment]
:+1: to what @adelbertc suggest, would make things easier indeed.

Great stuff folks

**Comment by adelbertc** [comment]
New PR made here: https://github.com/ceedubs/sbt-ctags/pull/11

So I gave it a bit more thought and here's what I've come up with:

1. Putting tag files in it's own project subdirectory makes things more clean organizationally, but can become annoying having to say, delete .tag files in each directory one at a time. Also, if you start the current working directory in one of the project subdirectories, then Vim (or other editors?) won't pick up other project tag files. If sub-project A depends on B, the generated tags for A doesn't include B so jump-to-definition doesn't like it. It also becomes tricky to try to gather the tag files from other subprojects.

2. Putting all the tag files in the root project directory under different names makes things a bit messier, but it's easier to manage manually, and because it seems editors will keep going up the folder tree until it hits a .tag file, we can still start the current working directory in one of the project subdirectories while still getting the .tag files for everyone.

Not sure if those points are coherent.. having a hard time explaining at the moment.

tl;dr My current theory is that putting all the tag files in the root project directory under different names seems to be the way to go.
